### PR TITLE
Add Spanish Fan Club DS

### PR DESCRIPTION
**Pull request by GuardedLolz**
### Prerequisites

* [X] I have read and understood the [contribution guidelines](CONTRIBUTING.md)
* [X] This pull request follows the code style of the project
* [X] I have tested this feature thoroughly WITH THE USE of the load-all-sounds starter
  * capture
* [X] This pull request is merging into **`dev`**

### Changes Proposed in this Pull Request
Add Spanish Fan Club DS
Add landing sounds for all three Fan Club DS languages

### CODE REVIEW

File: core/assets/sounds/cues/fanClubEn/data.json [hunk 1]
```diff
@@ -92,6 +92,11 @@
       "name": "audience - I suppose - 4"
     },
     {
+      "id": "fanClubEn/landing",
+      "duration": 1,
+      "name": "audience - i suppose - landing"
```
Comment (chrislo27) [inline core/assets/sounds/cues/fanClubEn/data.json:97]: Capitalize the `i`.

**Review by chrislo27 — CHANGES_REQUESTED** [review]
"yalose!" should be "¡ya lo sé!" (correct accents)

**Comment by GuardedLolz** [comment]
It really didn't matter if you merged my PR or not; I have MUCH bigger plans. Basically, what I am going to do is make RHRE as good as I can make it. See ya around.

**Comment by chrislo27** [comment]
I personally prefer smaller pull requests so it's easier to review. You have a higher chance at getting a large pull request rejected because of how much work it would be to review it all at once.

Please keep this in consideration.

**Pull request closed without merging**